concepts/symbols/about.md

```diff
@@ -0,0 +1,80 @@
+# About
+
+[Symbols][symbols] are named identifiers that can be used to refer to a value.
+Symbols are created through a symbol literal, which is by prefixing a name with a `:` character, e.g. `:foo`.
+They also allow for being written as strings, e.g. `:"foo"`, which allows for example spaces in the name.
+
+```ruby
+:foo # => :foo
+:"foo boo" # => :"foo boo"
+```
+
+Symbols are used in many places in the language, including as keys in hashes, to represent method names and variable names.
+
+## Identifier
+
+What makes symbols different from strings is that they are identifiers, and not data or text.
```

Comment:
They are not "text", but then what is "text"?  In Ruby, it is not a thing.

I think they are, and can be used as, data.  When mentoring for RubyLearning I would say "symbols are used for their identity, the data is the identity is the symbol".

Fabio Akita wrote:

> You can also think of `:id` as meaning the name of the variable `id`, and plain `id` as meaning the value of the variable.

Reply:
"[Symbol](https://ruby-doc.org/3.2.2/Symbol.html) objects are different from [String](https://ruby-doc.org/3.2.2/String.html) objects in that [Symbol](https://ruby-doc.org/3.2.2/Symbol.html) objects represent identifiers, while [String](https://ruby-doc.org/3.2.2/String.html) objects represent text or data." - Ruby Docs, Symbols

I could likely change the wording a bit and say:

> What makes symbols different from strings is that they are identifiers, and do not represent data or text.

Reply:
I like that better.  And yeah, the Ruby Docs is right, it does not say that Ruby has "text" only that String objects represent "text".

I would have worded it a little more strongly in the docs, as Symbol objects are identifiers, they do not represent identifiers.  They do represent the "non-symbol" names of those things identified though. You can see this in action when you do something like `Object.methods` and see the symbols that are the identifiers that represent the names of those methods.